.semaphore/semaphore.yml

```diff
@@ -140,7 +144,7 @@ blocks:
     - name: "Typha: UT and FV tests"
       commands:
       - cd typha
-      - make ci EXCEPT=k8sfv-test
+      - "../.semaphore/run-and-monitor make-ci.log make ci EXCEPT=k8sfv-test"
```

Comment:
Looks like not because some lines don't have the quotes.

e.g., 

```
- ../.semaphore/run-and-monitor fv-${SEMAPHORE_JOB_INDEX}.log make fv FV_BATCHES_TO_RUN="${SEMAPHORE_JOB_INDEX}" FV_NUM_BATCHES=${SEMAPHORE_JOB_COUNT}
```

Personally prefer without but either way we should be consistent